packages/ui-react/src/containers/AccountModal/forms/Login.tsx

```diff
@@ -29,13 +29,21 @@ const Login: React.FC<Props> = ({ messageKey }: Props) => {
 
   const queryClient = useQueryClient();
 
+  // temporary state to show success message
+  const [successState, setSuccessState] = React.useState(false);
```

Comment:
IMO, the state is not temporary unless you're still working on a better solution for keeping the successful state.

``` suggestion
  // state to show the success message
  const [successState, setSuccessState] = useState(false);
```

Reply:
I would like to add to this that code comments should not be necessary if we can let the code speak for itself. So, the suggestion Christiaan has in his example code is good to apply:

`const [announcement, setAnnouncement] = useState(false);`

Reply:
I think `state` is unnecessary in the name of a state, and perhaps adding `is` would make it a bit more clear that it's a boolean when scanning the code:
`const [isSuccess, setIsSucccess] = useState(false);`, or use the toggle:
`const [isSuccess, toggleSuccess] = useToggle(false);`

Reply:
I've removed the state in favor of the AriaAnnouncer, the state was in this case indeed temporary and would be removed on the coming commits. 

Thanks for the suggestions everyone!